js/huobipro.js

```diff
@@ -179,11 +179,11 @@ module.exports = class huobipro extends Exchange {
         let open = this.safeFloat (ticker, 'open');
         let close = this.safeFloat (ticker, 'close');
         let change = undefined;
-        let percentage = undefined;
+        let ratio = undefined;
         if ((typeof open !== 'undefined') && (typeof close !== 'undefined')) {
             change = close - open;
             if ((typeof last !== 'undefined') && (last > 0))
-                percentage = (change / last) * 100;
+                ratio = (change / open) * 100;
```

Comment:
We have several options for `ratio`:
- new / old
- change / old
- change / open * 100
- remove `ratio` due to ambiguity?
- other?

What do you think?

Reply:
If we speak about "percentage change" I believe that its defacto widespread meaning is `(new - old) / old`. That's what I've been taught at school a while ago, maybe something changed since that time )

"Ratio" is not as precise because it implies that the units of measure are not necessarily the same (e.g. "50 mph").

> What do you think? 

I'm actually fine if we drop it ) It's a rare breed, only few exchanges have:
```
js/bibox.js:            'percentage': this.safeString (ticker, 'percent'),
js/binance.js:            'percentage': this.safeFloat (ticker, 'priceChangePercent'),
js/bitfinex2.js:            'percentage': ticker[length - 5],
js/bitlish.js:            'percentage': this.safeFloat (ticker, 'prc'),
js/paymium.js:            'percentage': this.safeFloat (ticker, 'variation'),
```

> (no worries, will merge anyway)

I have no worries about my commits, really )